src/libraries/System.Runtime/tests/System/Reflection/ModuleTests.cs

```diff
@@ -348,13 +347,25 @@ public void ResolveStringFail(int token)
             });
         }
 
-        [Theory]
-        [MemberData(nameof(Types))]
-        [MemberData(nameof(Methods))]
-        [MemberData(nameof(Fields))]
-        public void ResolveMember(MemberInfo member)
+        [Fact]
+        public void ResolveTypesByMemberInfo()
+        {
+            foreach(MemberInfo mi in Types)
```

Comment:
Nittiest nit possible? Inconsistent spacing on the `foreach` statements.

Reply:
Since it's a nittiest nit, and validation's done, I'll fix that in the next change.